src/Servers/IIS/IISIntegration/src/IISDefaults.cs

```diff
@@ -3,10 +3,22 @@
 
 namespace Microsoft.AspNetCore.Server.IISIntegration
 {
+    /// <summary>
+    /// Defaults to configure IIS Out-Of-Process with.
```

Comment:
```suggestion
    /// String constants used to configure IIS Out-Of-Process
```

or something like that?